dotnet/src/IntegrationTests/Connectors/Anthropic/TestBase.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.Anthropic;
+using Xunit.Abstractions;
+
+namespace SemanticKernel.IntegrationTests.Connectors.Anthropic;
+
+public abstract class TestBase(ITestOutputHelper output)
+{
+    private readonly IConfigurationRoot _configuration = new ConfigurationBuilder()
+        .AddJsonFile(path: "testsettings.json", optional: false, reloadOnChange: true)
```

Comment:
Settings are optional as we prefer UserSecrets.

```suggestion
        .AddJsonFile(path: "testsettings.json", optional: true, reloadOnChange: true)
```